.github/workflows/modernisation-platform-restart-service.yaml

```diff
@@ -0,0 +1,36 @@
+name: Modernisation Platform Restart Service
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: 'Environment to restart Jitbit in'
+        required: true
+        default: 'development'
+        type: choice
+        options:
+        - development
+        - test
+        - preproduction
+        - production
+jobs:
+  restart-jitbit:
+    name: Restart Jitbit - ${{ inputs.environment }}
+    runs-on: ubuntu-latest
+    environment: ${{ inputs.environment }}
+    env:
+      SERVICE_TASK_NAME: hmpps-${{ inputs.environment }}-delius-jitbit
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-access-key-id: ${{ secrets.CICD_ACCESS_KEY }}
```

Comment:
Shouldn't these secrets differ depending on the environment chosen?

Reply:
no, this will use environment scoped secrets. Each environment has them that corresponds to the correct account